### ISSUE TITLE
V5.1 Inner handle cosmetic layer shift?

### ISSUE THREAD

**Issue by KookyMawnster** [OUTOFDARTSSHOP/Rival-Juno-Blaster#3]
Hi.  Printed V5 of inner handle at first, then saw V5.1 so printed that.  There seems to be a slight offset in the layers in V5.1 that was not there in V5.  It is apparent on the right side of the trigger guard, and IMO is cosmetic.  I printed 2 copies in different rotations and the layer shift is in both.

**Comment by Turokhunter** [comment]
Can you post an image so I can see what is happening? 5.1 change was to make it a bit thinner to slide in.

**Comment by KookyMawnster** [comment]
Hi Tarik,

V5 is in back, V5.1 in front. It is very subtle.

Love your design! I am printing it in black and yellow and call it Juno "Stinger".

Regards,
Mark

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by KookyMawnster** [comment]
Sorry here is the picture of the InnerHandle. And some pictures of my completed Juno, which I call "Stinger."
![20231214_183925](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/153342260/1100d128-5768-4d04-bedf-06c7e95eeaae)
![20231208_214623](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/153342260/4b10f853-61e6-4a12-8d6f-f655c0fb0bca)
![20231208_215028](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/153342260/4ba408a1-6dd1-45be-bbd2-d65a50e400ad)
![20231208_215147](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/153342260/bf7ec5fd-55f0-4154-80cd-eab16359a4a5)

**Comment by Turokhunter** [comment]
Nice, I was wondering if someone would take advantage of the center area and change up the color.  Is the layer shift on the same side as the flat plate? like it is on the Z-height location?

**Comment by KookyMawnster** [comment]
If I understand your question, I will answer yes.  I have not tried to import the .stl into a CAD program, but it "appears" as though the base object upon which the other elements are built upon had shifted down in the Y-direction as oriented on the plate.  Although the "shift" is only apparent in the trigger guard area as shown and NOT in any other areas of the InnerHandle.

**Comment by KookyMawnster** [comment]
After comparing the V5 and V5.1 prints more closely, another way of describing this is that it appears the oval "hole" of the trigger guard area was made slightly bigger in V5.1.  The shift goes all the way around the inside of the trigger guard where the "base layer" meets the other elements printed on top of it.

**Comment by Turokhunter** [comment]
When you printed it, did you use a 3mf file or import the STL file directly into a slicer?
I'm not seeing anything in the STL viewer.
![image](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/7236305/95d914b3-4180-4d4c-b583-c05fd69d409f)

What about when you look at through the slicer? I'm not seeing it in mine, but I'm interested to know if you are.

![image](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/7236305/bc8b8452-1e3e-447d-acc9-712d2ad30b8e)

**Comment by KookyMawnster** [comment]
Oh hey I'm seeing it in the slicer......
![IHV5_2](https://github.com/OUTOFDARTSSHOP/Rival-Juno-Blaster/assets/153342260/fb68582c-c775-48bf-a5fb-0b3bc7da242b)
It's printer settings then.  I went back to default standard and I don't see that anomaly anymore.  I even modified the settings from default again to what I thought I did and I'm not replicating it anymore.  I'm at a loss what setting(s) is causing that.
